eventmesh-dashboard-common/pom.xml

```diff
@@ -40,6 +40,39 @@
             <artifactId>fastjson2</artifactId>
             <version>2.0.40</version>
         </dependency>
+        <!-- storage redis client -->
+        <dependency>
+            <groupId>io.lettuce</groupId>
+            <artifactId>lettuce-core</artifactId>
+        </dependency>
+        <!-- Meta - nacos client -->
+        <dependency>
+            <groupId>com.alibaba.nacos</groupId>
+            <artifactId>nacos-client</artifactId>
+            <version>${nacos.version}</version>
+        </dependency>
+        <!-- Unit Test -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+        </dependency>
```

Comment:
`spring-boot-starter-test` artifact should have a test scope. Leave it in `console` will be better.

Reply:
soga